### PR TITLE
Blog post for 2020 steering election results

### PR DESCRIPTION
**Pull request by rvennam**
Blog post for 2020 steering election results

cc: @istio/steering-committee 

[ ] Configuration Infrastructure
[X ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/blog/2020/steering-election-results/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Last month, we [announced a revision to our Steering Committee charter](../steering-changes/), opening up governance roles to more contributors and community members. The Steering Committee now consists of 9 proportionally-allocated Contribution Seats, and 4 elected Community Seats.
+
+We have now concluded our [inaugural election](https://github.com/istio/community/tree/master/steering/elections/2020) for the Community Seats, and we're excited to welcome the following new members to the Committee:
+
+- [Neeraj Poddar](https://github.com/istio/community/blob/master/steering/elections/2020/nrjpoddar.md) (Aspen Mesh)
+- [Zack Butcher](https://github.com/istio/community/blob/master/steering/elections/2020/zackbutcher.md) (Tetrate)
+- [Christian Posta](https://github.com/istio/community/blob/master/steering/elections/2020/ceposta.md) (Solo.io)
+- [Zhonghu Xu](https://github.com/istio/community/blob/master/steering/elections/2020/hzxuzhonghu.md) (Huawei)
+
+They join Contribution Seat holders from Google, IBM/Red Hat and Salesforce. We now have representation from 7 organizations on Steering, reflecting the breadth of our contributor ecosystem.
```
Comment (hzxuzhonghu) [inline content/en/blog/2020/steering-election-results/index.md:19]: s/representation/representatives ?

**Comment by linsun** [comment]
LGTM - thanks @rvennam!

Congrats and welcome to our newest steering members!

File: content/en/blog/2020/steering-election-results/index.md (hunk 1 above)
Comment (rvennam) [reply content/en/blog/2020/steering-election-results/index.md:19]: I like representation better as it puts the focus on the organizations. The point of this statement is to highlight the corporate diversity.

File: content/en/blog/2020/steering-election-results/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: "2020 Steering Committee Election Results"
+subtitle: Welcome to our newest Steering Committee members
```
Comment (craigbox) [inline content/en/blog/2020/steering-election-results/index.md:3]:
```suggestion
subtitle: Welcome to our newest members
```

**Review by craigbox — APPROVED** [review]
LGTM with one change.

**Review by smawson — APPROVED** [review]
Short and sweet, I like it :)

**Review by frankbu — APPROVED** [review]

**Pull request merged**